dashboard/db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2023_03_09_041659) do
+ActiveRecord::Schema.define(version: 2023_03_13_222451) do
```

Comment:
@davidsbailey , how are conflicts between migrations resolved when the migration was generated much earlier than the time at which change is merged. Especially, if there are newer migrations checked in since the engineer forked staging branch.

Reply:
good question. :-) I usually resolve the conflict arbitrarily, and then rerun `rake db:migrate` and go with whatever it generates.

Reply:
I'm getting the same sense reading [this article](https://natemacinnes.github.io/how-to-handle-schema-conflicts-when-rebasing-git-rails.html), that running rake db:migrate will generate the most recent schema.rb with the appropriate version consolidating the conflicting migrations. I guess this should work in most cases, unless the conflicting migration and the one in feature branch do a DB operation itself which has potential conflicts. I'm assuming that's a fairly rare scenario.